### PR TITLE
Fix documentation for createLinkerInput owner

### PR DESCRIPTION
**Pull request by deeglaze**
The existing text is a clear copy/paste error. My proposed replacement is the most sensible explanation I can give for this field, given my lack of context for this component.

Suggested reviewer: plf@

### CODE REVIEW

File: src/main/java/com/google/devtools/build/lib/skylarkbuildapi/cpp/CcModuleApi.java [hunk 1]
```diff
@@ -633,7 +633,7 @@ LibraryToLinkT createLibraryLinkerInput(
       parameters = {
         @Param(
             name = "owner",
-            doc = "List of <code>LibraryToLink</code>.",
+            doc = "The label of the target that produced any files used in this input.",
```
Comment (ulfjack) [inline src/main/java/com/google/devtools/build/lib/skylarkbuildapi/cpp/CcModuleApi.java:636]: I think it should probably be "all" rather than "any"

**Comment by ulfjack** [comment]
@oquenchil

**Comment by ulfjack** [comment]
@aiuto It's odd that this hasn't been looked at for almost a week.

File: src/main/java/com/google/devtools/build/lib/skylarkbuildapi/cpp/CcModuleApi.java (hunk 1 above)
Comment (deeglaze) [reply src/main/java/com/google/devtools/build/lib/skylarkbuildapi/cpp/CcModuleApi.java]: Done.

**Pull request closed without merging**